pkg/sql/sqlinstance/instancestorage/instancereader_test.go

```diff
@@ -175,6 +175,15 @@ func TestReader(t *testing.T) {
 			}
 			return nil
 		}
+		getInstancesUsingTxnWrapper := func() ([]sqlinstance.InstanceInfo, error) {
+			txn := s.DB().NewTxn(ctx, "testing GetAllInstancesUsingTxn")
```

Comment:
`NewTxn` is low-level and generally to be avoided. Instead you ought to use `s.DB().Txn`